extensions/amp-mega-menu/amp-mega-menu.md

```diff
@@ -0,0 +1,325 @@
+<table>
+  <tr>
+    <td width="40%"><strong>data-close-button-aria-label (optional)</strong></td>
+    <td>Optional attribute used to set ARIA label for the close buttons added for accessibility.</td>
+  </tr>
+  <tr>
+    <td width="40%"><strong>common attributes</strong></td>
+    <td>This element includes <a href="https://amp.dev/documentation/guides-and-tutorials/learn/common_attributes">common attributes</a> extended to AMP components.</td>
+  </tr>
+</table>
+
+## Styling
+
+The `<amp-mega-menu>` component can be styled with standard CSS.
+
+- By default, a white background is applied to the `<nav>` element and the expandable content elements.
+- When open, the content containers will fill the entire viewport width but can be overridden with the `left` and `width` properties.
+- The current state of the mega menu is exposed via the `open` attribute that is set on the `amp-mega-menu` tag when one of the menu items is expanded.
+- The `<li>` element with expanded content will also receive the `open` attribute, which developers can use to style its children.
+
+In the example below, we will customize the background color of the navigation bar, the appearance of menu buttons when opened, as well as the positioning of dropdown containers.
+
+[example playground="true" preview="top-frame" orientation="landscape"]
+```html
+<head>
+  <script async custom-element="amp-mega-menu" src="https://cdn.ampproject.org/v0/amp-mega-menu-0.1.js"></script>
+  <style amp-custom>
+    .title {
+      background-color: lightblue;
+      padding: 0.5em;
+      margin: 0;
+    }
+    amp-mega-menu nav {
+      background-color: lightgray;
+    }
+    amp-mega-menu .menu-item[open] > span {
+      background-color: white;
+    }
+    amp-mega-menu .menu-item[open] > div {
+      left: 10px;
+      right: 10px;
+      width: auto;
+    }
+  </style>
+</head>
+<body>
+  <header>
+    <h1 class="title">styling the amp-mega-menu</h1>
+    <amp-mega-menu height="30" layout="fixed-height">
+      <nav>
+        <ul>
+          <li class="menu-item">
+            <span role="button">List 1</span>
+            <div role="dialog">
+              <ol>
+                <li>item 1.1</li>
+                <li>item 1.2</li>
+                <li>item 1.3</li>
+              </ol>
+            </div>
+          </li>
+          <li class="menu-item">
+            <span role="button">List 2</span>
+            <div role="dialog">
+              <ol>
+                <li>item 2.1</li>
+                <li>item 2.2</li>
+                <li>item 2.3</li>
+              </ol>
+            </div>
+          </li>
+        </ul>
+      </nav>
+    </amp-mega-menu>
+  </header>
+</body>
+```
+[/example]
+
+## Accessibility
+
+`<amp-mega-menu>` assigns the following ARIA attributes on the children of each expandable menu item. These attributes are used by screen reader technology to associate a button with its togglable container as well as to trap focus inside the container once opened.
```

Comment:
Avoid passive voice, update second sentence:
Screen readers use these attributes to associate buttons with toggleable container and trap focus inside opened containers.